roles/tackle/defaults/main.yml

```diff
@@ -176,7 +176,7 @@ provider_java_component_name: "extension"
 provider_java_container_limits_cpu: "1"
 provider_java_container_limits_memory: "2Gi"
 provider_java_container_requests_cpu: "1"
-provider_java_container_requests_memory: "2Gi"
+provider_java_container_requests_memory: "1Gi"
```

Comment:
This should stay at 2GI for the request the JVM will use 70% of max memory, and means that we can assume that it will use the full amount IMO. It will be safer to set this limit equal to the requests